src/Exceptions/HttpUpgradeRequired.php

```diff
@@ -8,4 +8,3 @@ class HttpUpgradeRequired extends HttpException {
     public const DESCRIPTION = "Indicates that the server refuses to perform the request using the current protocol but might be willing to do so after the client upgrades to a different protocol.";
     public const HREF = "https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/426";
 }
-
```

Comment:
Removed unix line ending - ALWAYS have a unix line ending, it can cause very weird issues.

Just use https://marketplace.visualstudio.com/items?itemName=jkiviluoto.tws you can use this to add them automatically. It also removes trailing whitespaces